packages/parser/test/css_parser_test.re

```diff
@@ -4,14 +4,35 @@ let parse = input => {
   let pos = Some(Lexing.dummy_pos);
   switch (Driver.parse_stylesheet(~pos, input)) {
   | Ok(ast) => Ok(ast)
-  | Error((_loc, msg)) => Error(msg)
+  | Error((loc, msg)) =>
+    let pos = loc.Css_types.loc_start;
+    let curr_pos = pos.pos_cnum;
+    let lnum = pos.pos_lnum;
+    let pos_bol = pos.pos_bol;
+    let err =
+      Printf.sprintf(
+        "%s on line %i at position %i",
+        msg,
+        lnum,
+        curr_pos - pos_bol,
```

Comment:
Do you think is worth it add this level of detail on the error reporting by the ppx?

Reply:
Yes, I think it is for clarity.